scrapper/controllers/miner.py

```diff
@@ -42,13 +45,14 @@ def mine_user(user_name, refresh):
     '''
     user = User.query.filter(User.username==user_name).first()
```

Comment:
It's time to make this task dumber. It was given a order, to gather information, and that's what it should do.